internal/flags/flags.go

```diff
@@ -152,22 +153,31 @@ func RegisterSystemFlags(rootCmd *cobra.Command) {
 		"",
 		viper.GetString("WATCHTOWER_HTTP_API_TOKEN"),
 		"Sets an authentication token to HTTP API requests.")
+
 	flags.BoolP(
 		"http-api-periodic-polls",
 		"",
 		viper.GetBool("WATCHTOWER_HTTP_API_PERIODIC_POLLS"),
 		"Also run periodic updates (specified with --interval and --schedule) if HTTP API is enabled")
+
 	// https://no-color.org/
 	flags.BoolP(
 		"no-color",
 		"",
 		viper.IsSet("NO_COLOR"),
 		"Disable ANSI color escape codes in log output")
+
 	flags.StringP(
 		"scope",
 		"",
 		viper.GetString("WATCHTOWER_SCOPE"),
 		"Defines a monitoring scope for the Watchtower instance.")
+
+	flags.BoolP(
+		"porcelain",
+		"P",
+		viper.GetBool("WATCHTOWER_PORCELAIN"),
+		"Write session results to stdout (alias for --notification-url logger:// --notification-log-stdout)")
```

Comment:
```suggestion
	flags.StringP(
		"porcelain",
		"P",
		viper.GetString("WATCHTOWER_PORCELAIN"),
		"Write session results in a stable versioned format to stdout. Only possible argument is v1")
```
...or scrap versioning.

Reply:
Added the version argument.